task-ballerina/tests/scheduler_timer_test.bal

```diff
@@ -98,12 +106,16 @@ service object {} service4 = service object {
 
 @test:Config {}
 function testTaskTimerWithSameServices() {
-    Scheduler timerWithMultipleServices = new ({intervalInMillis: 1000});
-    checkpanic timerWithMultipleServices.attach(service4);
-    checkpanic timerWithMultipleServices.start();
-    runtime:sleep(1.5);
-    checkpanic timerWithMultipleServices.attach(service4);
-    runtime:sleep(2.7);
-    checkpanic timerWithMultipleServices.stop();
-    test:assertTrue(fourthTimerServiceTriggered, msg = "Expected value mismatched");
+    Scheduler|SchedulerError timerWithMultipleServices = new ({intervalInMillis: 1000});
+    if (timerWithMultipleServices is Scheduler) {
+        var result = timerWithMultipleServices.attach(service4);
+        result = timerWithMultipleServices.start();
+        runtime:sleep(1.5);
+        result = timerWithMultipleServices.attach(service4);
+        runtime:sleep(2.7);
+        result = timerWithMultipleServices.stop();
+        test:assertTrue(fourthTimerServiceTriggered, msg = "Expected value mismatched");
+    } else {
+        test:assertFail("Test failed due to an error in the creating of the scheduler.");
+    }
 }
```

Comment:
Instead of checking the error here, can't we change the function signature to return `error?` and then use check directly?

Reply:
Fixed in d6d1e47